src/components/box/_box.scss

```diff
@@ -1,14 +1,19 @@
 $boxBorderColor: $graySecondary;
 $boxBorderRadius: 22px;
+$boxBorderSize: rhythm(1/8);
 
 @if ($include-html) {
 
   .mint-box {
     @include component;
-
+    min-height: rhythm(3);
     border-radius: $boxBorderRadius;
-    padding: $layoutDefaultPadding;
-    border: 2px solid $boxBorderColor;
+    border: $boxBorderSize solid $boxBorderColor;
+    padding: (rhythm(1/2) - $borderSize) 0 (rhythm(1/2) - $borderSize) 1/2 * $layoutDefaultPadding;
+
+    &__hole {
+      margin: rhythm(1/2) 1/2 * $layoutDefaultPadding rhythm(1/2) 0;
```

Comment:
This gives the margin from top, bottom and right sides.

Half-margin is used, as other half goes to the container paddings. 

In combination, this allows to have separate content holes in a **single line with wrapping**, snapped to vertical grid (see image in description)